repo/packages/A/artifactory-primary/1/marathon.json.mustache

```diff
@@ -0,0 +1,90 @@
+{
+  "id": "{{artifactory-primary.framework-name}}",
+  "cpus": {{artifactory-primary.cpus}},
+  "mem": {{artifactory-primary.mem}},
+  {{^artifactory-primary.high-availability.enabled}}
+  "instances": 1,
+  {{/artifactory-primary.high-availability.enabled}}
+  {{#artifactory-primary.high-availability.enabled}}
+  "instances": {{artifactory-primary.high-availability.nodes}},
+  {{/artifactory-primary.high-availability.enabled}}
+   "container": {
+       "type": "DOCKER",
+       "docker": {
+           {{^artifactory-primary.high-availability.enabled}}
+           "image": "{{resource.assets.container.docker.d2284754a53c}}",
+           {{/artifactory-primary.high-availability.enabled}}
+           {{#artifactory-primary.high-availability.enabled}}
+           "image": "{{resource.assets.container.docker.a41b154c2257}}",
+           {{/artifactory-primary.high-availability.enabled}}
+           "network": "HOST",
+            "forcePullImage":true
+       },
+       "volumes": [
+          {{^artifactory-primary.high-availability.enabled}}
+          {
+          "containerPath": "/var/opt/jfrog/artifactory/data",
+          "hostPath": "{{artifactory-primary.artifactory-home}}/{{artifactory-primary.framework-name}}/data",
+          "mode": "RW"
+          },
+          {
+             "containerPath": "/var/opt/jfrog/artifactory/logs",
+             "hostPath": "{{artifactory-primary.artifactory-home}}/{{artifactory-primary.framework-name}}/logs",
+             "mode": "RW"
+          },
+          {
+             "containerPath": "/var/opt/jfrog/artifactory/backup",
+             "hostPath": "{{artifactory-primary.artifactory-home}}/{{artifactory-primary.framework-name}}/backup",
+             "mode": "RW"
+          }
+          {{/artifactory-primary.high-availability.enabled}}
+          {{#artifactory-primary.high-availability.enabled}}
+          {
+             "containerPath": "/var/opt/jfrog/cluster",
+             "hostPath": "{{artifactory-primary.high-availability.cluster-home}}/cluster-home",
+             "mode": "RW"
+          },
+          {
+          "containerPath": "/var/opt/jfrog/artifactory/data",
+          "hostPath": "{{artifactory-primary.high-availability.cluster-home}}/{{artifactory-primary.framework-name}}/data",
+          "mode": "RW"
+          },
+          {
+          "containerPath": "/var/opt/jfrog/artifactory/logs",
+          "hostPath": "{{artifactory-primary.high-availability.cluster-home}}/{{artifactory-primary.framework-name}}/logs",
+          "mode": "RW"
+          },
+          {
+          "containerPath": "/var/opt/jfrog/artifactory/backup",
+          "hostPath": "{{artifactory-primary.high-availability.cluster-home}}/{{artifactory-primary.framework-name}}/backup",
+          "mode": "RW"
+          }
+          {{/artifactory-primary.high-availability.enabled}}
+       ]
+   },
+  "labels": {
+      "HAPROXY_GROUP":"external"
```

Comment:
You should allow the user to specify an optional HAPROXY_0_VHOST too. For example:
```    {{#artifactory-primary.virtual-host}}
    "HAPROXY_GROUP":"external",
    "HAPROXY_0_VHOST":"{{artifactory-primary.virtual-host}}"
    {{/artifactory-primary.virtual-host}}```

You would also add the following to your config.json:
```
        "virtual-host": {
            "description": "The virtual host address to configure for integration with Marathon-lb.",
            "type": "string"
        },```